core/eth/tx.go

```diff
@@ -588,7 +589,7 @@ func (t *Transactor) CalculateOperatorChurnApprovalDigestHash(
 	}
 	return t.Bindings.RegistryCoordinator.CalculateOperatorChurnApprovalDigestHash(&bind.CallOpts{
 		Context: ctx,
-	}, operatorId, opKickParams, salt, expiry)
+	}, operatorAddress, operatorId, opKickParams, salt, expiry)
```

Comment:
@0x0aa0 This is where we depend on the onchain change

Reply:
lgtm!